tests/TestRunner/local_cluster.py

```diff
@@ -250,10 +254,15 @@ def save_config(self):
         with open(new_conf_file, "x") as f:
             conf_template = LocalCluster.configuration_template
             bg_knob_line = ""
+            encrypt_knob_line1 = ""
+            encrypt_knob_line2 = ""
             if self.use_legacy_conf_syntax:
                 conf_template = conf_template.replace("-", "_")
             if self.blob_granules_enabled:
                 bg_knob_line = "knob_bg_url=file://" + str(self.data) + "/fdbblob/"
+            if self.enable_encryption_at_rest:
+                encrypt_knob_line1 = "knob_enable_encryption=true"
```

Comment:
May be this KNOB would be deprecated soon as the `encryption_at_rest` database config would govern the encryption properties for the cluster.

Reply:
Yeah since it was needed for now and we're deprecating and then cleaning it up, I assume this would just get cleaned up in whatever PR does so.